index.d.ts

```diff
@@ -78,15 +106,25 @@ export interface TestInterface<Context = {}> {
 	(title: string, macro: Macro<Context> | Macro<Context>[], ...args: Array<any>): void;
 	(macro: Macro<Context> | Macro<Context>[], ...args: Array<any>): void;
 
+	// Test hook that runs after all tests are done
```

Comment:
Try and describe what the modifier *does*, rather than what it *is*. E.g. here "Specify hooks that run once all tests have completed". Or below, "Skip this test."